Runtimes/Core/core/CMakeLists.txt

```diff
@@ -325,9 +325,12 @@ elseif(NOT APPLE AND NOT LINUX AND NOT ANDROID AND UNIX)
   target_link_libraries(swiftCore PRIVATE "${EXECINFO_LIBRARY}")
 endif()
 
-install(TARGETS swiftCore)
+install(TARGETS swiftCore
+  ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}/$<$<NOT:$<PLATFORM_ID:Darwin>>:swift$<$<NOT:$<BOOL:${BUILD_SHARED_LIBS}>>:_static>/${SwiftCore_PLATFORM_SUBDIR}/${SwiftCore_ARCH_SUBDIR}>"
+  LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}/$<$<NOT:$<PLATFORM_ID:Darwin>>:swift$<$<NOT:$<BOOL:${BUILD_SHARED_LIBS}>>:_static>/${SwiftCore_PLATFORM_SUBDIR}/${SwiftCore_ARCH_SUBDIR}>"
```

Comment:
The `<platform>/<arch>` install directory is not correct for all build configurations either. The macOS builds, for instance, should be to `/usr/lib/swift/libswiftCore.dylib`, not `/usr/lib/swift/macosx/arm64`. Please add an option to disable the including the extra directories. Same for the swift module.

Reply:
The `$<NOT:$<PLATFORM_ID:Darwin>>` does that though. Is there a reason to generalise this into an option?